src/fosslight_source/_parsing_scanoss_file.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 LG Electronics Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import logging
+import re
+import fosslight_util.constant as constant
+from ._license_matched import MatchedLicense
+from os.path import dirname
+
+logger = logging.getLogger(constant.LOGGER_NAME)
+_replace_word = ["-only", "-old-style", "-or-later", "licenseref-scancode-"]
+_exclude_filename = ["changelog", "config.guess", "config.sub",
+                     "config.h.in", "changes", "ltmain.sh",
+                     "aclocal.m4", "configure", "configure.ac",
+                     "depcomp", "compile", "missing", "libtool.m4",
+                     "makefile"]
+_exclude_directory = ["test", "tests", "doc", "docs"]
+_exclude_directory = [os.path.sep + dir_name +
+                      os.path.sep for dir_name in _exclude_directory]
+_exclude_directory.append("/.")
+
+
+class ScanOssItem:
```

Comment:
If there is little difference between class ScanCodeItem of _parsing_scancode_file_item.py and this class, separate class ScanCodeItem of _parsing_scancode_file_item.py into another python file and use it in common.

Reply:
Separate the class from _parsing_scancode_file_item.py and _parsing_scanoss_file.py to _scan_item.py
And merged convert_scanoss.py into convert_scancode.py. (you might want to change the filename)